src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -69,7 +69,7 @@ object Boot extends App with LazyLogging {
 
         val googleDirDaos = (googleServicesConfig.adminSdkServiceAccounts match {
           case None => NonEmptyList.one(Pem(WorkbenchEmail(googleServicesConfig.serviceAccountClientId), new File(googleServicesConfig.pemFile), Option(googleServicesConfig.subEmail)))
-          case Some(accounts) => accounts.map(account => Json(account.json))
+          case Some(accounts) => accounts.map(account => Json(account.json, Option(googleServicesConfig.subEmail)))
```

Comment:
if `googleServicesConfig.subEmail` will not be `null`, shall we just use `Some`?

Reply:
it certainly should not be null but we follow the practice of using Option to avoid the possibility of Some(null).